src/components/Resume/index.js

```diff
@@ -558,7 +558,7 @@ const Resume = forwardRef(({ data }, ref) => {
                       </h4>
                     </div>
                     <div className={`${styles.profiledetails} pb-3`}>
-                      {profile?.description}
+                      <span style={{ whiteSpace: 'pre-line' }}>{profile?.description}</span>
```

Comment:
description should also support newlines.

Reply:
Done!